src/user/model.rs

```diff
@@ -92,6 +92,29 @@ impl User {
         })
     }
 
+    pub async fn delete_bot<'a, E>(
+        payload: &user_delete_bots_mutation::Payload,
+        executor: E,
+    ) -> Result<(), sqlx::Error>
+    where
+        E: Executor<'a>,
+    {
+        let mut transaction = executor.begin().await?;
+
+        for user_id in &payload.user_ids {
+            sqlx::query!(
+                r#"DELETE FROM uuid WHERE id = ? AND discriminator = 'user'"#,
+                user_id,
+            )
+            .execute(&mut transaction)
+            .await?;
```

Comment:
This should return a MySqlQueryResult which has a method https://docs.rs/sqlx/0.5.9/sqlx/mysql/struct.MySqlQueryResult.html#method.rows_affected giving the number of rows affected. Can you have a check whether it is `1` and raise an error when it is not the case?